packages/core/test/source/at-test.ts

```diff
@@ -12,4 +12,11 @@ describe('at', () => {
     return collectEventsFor(t, at(t, expected))
       .then(eq([{ time: t, value: expected }]))
   })
+  it('is auto-curried', function () {
```

Comment:
It's ok not to test this.  The types already guarantee it.  For example, here's the type error we'd get if we wrote this same code without at being curried:

<img width="600" alt="Screen Shot 2024-03-07 at 6 56 02 PM" src="https://github.com/mostjs/core/assets/90518/36c678b5-d88d-4125-9707-9b5f47ea769e">